setup_ubuntu_local.sh

```diff
@@ -2,25 +2,24 @@
 
 set +o histexpand
 set -eu
-
+source .env
 #####
 # Install required software
 #####
-sudo apt-get update -y
+# sudo apt-get update -y
```

Comment:
You left this commented out in the bash script, even though it was only because apt wasn't updating Postgres for you. This should be uncommented.